### PR TITLE
New report issue history

### PR DESCRIPTION
**Pull request by vrenaville**
Sometimes our customer need a details of issues linked to our block hours.
So I create a report that list all email received and send on issues.

### CODE REVIEW

**Comment by mdietrichc2c** [comment]
:+1:

**Comment by jgrandguillaume** [comment]
LGTM, thanks for the contribs. I saw the report and it looks good ! :+1:

File: project_issue_block_report_webkit/data/header.xml [hunk 1]
```diff
@@ -23,12 +23,13 @@
...
-                <td style="text-align:center;font-size:10;width:350px;">${user.name}</td>
-                <td style="text-align:right;font-size:10;width:350px;">Page&nbsp;<span class="page"/></td>
-                <td style="text-align:left;font-size:10;width:30px">&nbsp;of&nbsp;<span class="topage"/></td>
+                <td style="text-align:left;font-size:10px;width:350px;">${formatLang( str(datetime.datetime.today()), date_time=True)}</td>
+                <td style="text-align:center;font-size:10px;width:350px;">${user.name}</td>
+                <td style="text-align:right;font-size:10px;width:350px;">Page&nbsp;<span class="page"/></td>
+                <td style="text-align:left;font-size:10px;width:30px">&nbsp;of&nbsp;<span class="topage"/></td>
             </tr>
         </table>
+        ${_debug or ''|n}
```
Comment (pedrobaeza) [inline project_issue_block_report_webkit/data/header.xml:32]: Remove debug

File: project_issue_block_report_webkit/data/header.xml [hunk 2]
```diff
@@ -53,6 +54,7 @@
             ${css}
         </style>
     </head>
+    ${_debug or ''|n}
```
Comment (pedrobaeza) [inline project_issue_block_report_webkit/data/header.xml:57]: Remove debug

**Comment by vrenaville** [comment]
@pedrobaeza I have keep the debug to be able to print the report in HTML, sometimes I need to generate the report in HTML format instead of PDF for large report.

**Comment by dreispt** [comment]
You have a pylint error:

```
project_issue_block_report_webkit/report/block_history_issue.py:77: [W0105(pointless-string-statement), project_issue_history._get_related_issue] String statement has no effect
```

**Comment by vrenaville** [comment]
@dreispt Thanks for the review, I have made the correction, Travis error is not related to my PR

**Pull request merged**